### DISCUSSION TITLE
Alpha channel slider

### DISCUSSION THREAD

**Discussion by jaimeef** [google/model-viewer#3729] [Q&A]
Hello!

I wonder if someone might be able to help me with a school project I am doing.. I have a model with three different objects, each with their own material. I would like to be able to include a slider which allows the viewer to adjust the alpha individually for each of the three objects (they overlay each other).

I can see the sliders here to change the roughness/metalness https://modelviewer.dev/examples/scenegraph/#changeMaterial but I wonder whether I can target each material individually to change the alpha in a similar manner.

Any tips or points in the right direction would be greatly appreciated!

**Comment by elalish** [comment]
Yes, that should work just fine. You may want `getMaterialByName` which you can find in the [material API](https://modelviewer.dev/examples/scenegraph/#pickMaterialExample). Keep in mind that rasterizers don't always do an amazing job of rendering stacks of transparent things.

**Reply by jaimeef** [reply]
Thank you so much! I have managed to call the materials and apply the alpha mode to blend, however I cannot see which function targets the Alpha Factor. I can only seem to change the mode, and the Alpha cutoff (which I don't think is what I want here
![Screen Shot 2022-08-24 at 10 57 39 am](https://user-images.githubusercontent.com/111841568/186293173-c38da362-2086-4445-a762-148e92c637b2.png)
)

**Reply by elalish** [reply]
alpha is the fourth element of `baseColorFactor` (since it can also be textured).

**Reply by jaimeef** [reply]
Ah oh course. Thank you very much!
